packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -182,13 +193,15 @@ async function generateBundleYaml(
   angularVersion?: string,
 ): Promise<void> {
   const runtimeEnvVars = addBundleYamlEnvVar(angularVersion);
+  const metadata = populateMetadata(angularVersion);
```

Comment:
ca we inline these two variables?

Reply:
You mean like `const runtimeEnvVars = addBundleYamlEnvVar(angularVersion), metadata = createMetadata(angularVersion);`?  I think this causes a lint error, have not discovered other ways to inline these two variables.

Reply:
Ahh got it thanks!